gcp/modules/locust/main.tf

```diff
@@ -18,9 +18,13 @@ resource "null_resource" "locust_link_tasks" {
     command = <<EOF
       mkdir -p ${var.charts_dir}/locust/tasks
       for FILE in tasks/*.py; do
-        echo "Creating link for $FILE"
-        ln -sf -T $PWD/$FILE ${var.charts_dir}/locust/$FILE
+        echo "Copying $FILE..."
+        cp -f $PWD/$FILE ${var.charts_dir}/locust/$FILE
       done
+      # Full write permissions are required so CI worker can remove those
+      # files from shared dir in case Locust module deployment fails
+      # and cleanup resource is never executed.
+      chmod -R a+w ${var.charts_dir}/locust/tasks
```

Comment:
I think we should use Docker Volumes for these files instead.

Reply:
I was hesitant if it worth the effort, but yeah, solution with docker volume looks cleaner anyway.